02-start.Rmd

```diff
@@ -2,24 +2,31 @@
 
 ## Getting an account
 
-You use a specially authenticated Github account to log into the platform, which you can set up as follows:
+You use a specially authenticated GitHub account to log into the platform, which you can set up as follows:
 
-### Sign up to Github
+### Sign up to GitHub
 If you do not already have an account, head to https://github.com/ and sign up. When signing up, use all the default options, and make sure you save recovery codes as instructed. It's good practice to choose a username that does not feature upper case characters.
 
-We recommend new users use their `@justice` email address.  If you already have a personal Github account, you may also use that.
+We recommend new users use their `@justice` email address.  If you already have a personal GitHub account, you may also use that.
 
 You'll receive an email with a verification link - you need to click this before using the platform.
 
-### Enable two-factor authentication on your Github account
-Once you've signed up to Github, you must enable two factor authentication on your Github account to enhance security. You can do this in your Github [settings page](https://github.com/settings/security).
+### Enable two-factor authentication on your GitHub account{#two-factor-authentication}
+Once you've signed up to GitHub, you must enable two-factor authentication (2FA) on your GitHub account to enhance security. You can do this from your GitHub [settings page](https://github.com/settings/security).
 
-You are given the option of doing this using SMS, or an authentication app such as [Authy](https://authy.com/download/) or [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en) . Choose whichever you prefer.
+You are given the option of doing this using SMS, or an authentication app, such as [Authy](https://authy.com/download/) or [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en). We recommend that you use __Authy__ for the following reasons:
 
-For step-by-step instructions, see the Github guidance for [authentication apps](https://help.github.com/articles/configuring-two-factor-authentication-via-a-totp-mobile-app/) or [SMS authentication](https://help.github.com/articles/configuring-two-factor-authentication-via-text-message/).
+1.  You can sync 2FA tokens across multiple devices, including mobiles, tablets and computers.
+2.  You can create encrypted recovery backups in case you lose your device.
+3.  You can use Touch ID, PIN codes and passwords to protect access to your 2FA tokens.
+4.  You can still access secure websites when you are offline.
```

Comment:
It's put a bit abstractly. How about: SMS codes don't work when you have no phone signal

Reply:
I agree. Updated in c81d959.